docs/console-api/daily-reports.mdx

```diff
@@ -57,50 +57,4 @@ import Rapidoc from "../../src/components/Rapidoc"
 
 - New columns may be added in the future.
 
-<br/>
-<hr/>
-
-# Deprecated: v1/get_daily_report
-
-## Authorization
-
-All requests must include the **Server-Side Secret Key** field in the header. This can can be created in the Project Settings on [console.statsig.com/api_keys](https://console.statsig.com/api_keys). This feature requires Pro or Enterprise Tier.
-
-:::info
-This is available for Enterprise contracts and needs to be enabled for you, once. Please reach out to our [support team](mailto:support@statsig.com), your sales contact, or via our slack channel if you want this enabled.
-:::
-
-## Responses
-
-All responses will contain a **url** field and a **date** field.
-
-- **url** is the URL containing the daily report.
-- **date** is the date of the currentlyavailable report.
-
-## Get Daily Reports
-
-You can [export your Pulse data](https://docs.statsig.com/pulse/export) via the console or using this API. This API supports two types of daily reports:
-
-1. [type=first_exposures](https://docs.statsig.com/pulse#first-exposures-file-description) - Provides a link to download your company's full set of "first exposures" across feature gates and experiments.
-2. [type=pulse_daily](https://docs.statsig.com/pulse#pulse-summary-and-daily-file-description) - Provides a link to download your company's daily experimental results. This data is a single day's results and a full experimental analysis requires the entire duration of the experiment.
-
-This feature is available to Pro and Enterprise tier customers.
-
-```bash
-curl \
-  --header "statsig-api-key: <YOUR-SERVER-SECRET-KEY>" \
-  --header "Content-Type: application/json" \
-  --request POST \
-  --data '{"type": "first_exposures" }' \
-  "https://api.statsig.com/v1/get_daily_report"
-```
-
-**Reponse JSON** :
-
-```json
-{
-  "url": <url to download CSV for the specified type>,
-  "date": <date in PST of data>,
-}
-```
-
+### Looking for our depricated [get_daily_reports](/console-api/daily-reports-depricated) endpoint? 
```

Comment:
"Looking for our depricated" -> (Deprecated: <link>)

Reply:
[fix spelling](https://github.com/statsig-io/docs/pull/718/commits/2446680a19c774eaae65f4c5d441c8489ce2d803)